### DISCUSSION TITLE
Option to rotate 128x64 display 90°

### DISCUSSION THREAD

**Discussion by tizzulo** [keirf/flashfloppy#510] [Ideas]
Curious if this can be implemented without too much trouble or space in the code. I'll see if I can play with the code and get it working. I have multiple devices which mount the floppy drive sideways, and the 128x64 seems it has the potential to have a nice rotated display for these vertically mounted devices.

**Comment by keirf** [comment]
It would be a very narrow display in that case? The font is 6x13 so it naturally gives plenty of line width at 128x64 aspect.
